api/runner/worker.go

```diff
@@ -296,6 +296,8 @@ func (hc *htfn) serve(ctx context.Context) {
 	}()
 
 	cfg := *hc.cfg
+	cfg.Env["FN_HOT"] = "true"
```

Comment:
I think hot is assumed if format is not `default` so probably don't need the extra var.